comp/etw/impl/etwSession.go

```diff
@@ -39,6 +44,10 @@ func (e *etwSession) ConfigureProvider(providerGUID windows.GUID, configurations
 }
 
 func (e *etwSession) EnableProvider(providerGUID windows.GUID) error {
+	// could also thrown an error here for improper api usage.
```

Comment:
I think adding warning if we have a logger component. Perhaps the error is better if no logging is done at this level.